### PR TITLE
Fix typo and formatting in README

### PR DESCRIPTION
**Pull request by aalaap**
A little MarkDown/HTML tidying along with a typo fix.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 Quirrel makes job queueing as easy as pie. It supports delayed jobs, fanout jobs, recurring jobs and CRON jobs.
 
-Quirrel values ...
+Quirrel values...
```
Comment (Skn0tt) [inline README.md:19]: According to https://www.grammarly.com/blog/ellipsis/, an ellipsis should be preceded by a space.
```suggestion
Quirrel values ...
```

File: README.md [hunk 2]
```diff
@@ -8,25 +8,23 @@
...
-The Task Queueing Solution for Serverless.
+  The Task Queueing Solution for Serverless.
 </p>
 
 - [Landing Page](https://quirrel.dev)
 - [Documentation](https://docs.quirrel.dev)
 
-Quirrel makes job queueing simple as cake.
-It supports delayed jobs, fanout jobs, recurring jobs and CRON jobs.
+Quirrel makes job queueing as easy as pie. It supports delayed jobs, fanout jobs, recurring jobs and CRON jobs.
```
Comment (Skn0tt) [inline README.md:17]: I'd actually like to keep "simple as cake" here. It always [reminds me of Portal 2](https://en.wikipedia.org/wiki/The_cake_is_a_lie) ;D Also, this is a catchphrase that's spread over a lot of different sources already, so it should be kept consistent :)

```suggestion
Quirrel makes job queueing simple as cake. It supports delayed jobs, fanout jobs, recurring jobs and CRON jobs.
```

File: README.md [hunk 3]
```diff
@@ -8,25 +8,23 @@
...
+- great DX (through [tailormade client libraries](https://docs.quirrel.dev/api/queue) & the [development UI](https://docs.quirrel.dev/development-ui))
+- privacy (through end-to-end encryption)
+- free software (it's MIT licensed!)
+- easy setup (through [hosted offering](https://quirrel.dev) and easy on-prem)
 
 ## Getting Started
 
-If you want to learn about Quirrel, ceck out [the tutorial](https://dev.to/quirrel/building-a-water-drinking-reminder-with-next-js-and-quirrel-1ckj)!
-
+If you want to learn about Quirrel, check out [the tutorial](https://dev.to/quirrel/building-a-water-drinking-reminder-with-next-js-and-quirrel-1ckj)!
```
Comment (Skn0tt) [inline README.md:28]: Good catch!

**Review by Skn0tt — CHANGES_REQUESTED** [review]
Thank you for the spell & style check! I like the removal of the list ellipsis. Left some comments :)

File: README.md (hunk 1 above)
Comment (aalaap) [reply README.md:19]: You learn something new each day!

File: README.md (hunk 2 above)
Comment (aalaap) [reply README.md:17]: But the cake was a lie!

**Comment by Skn0tt** [comment]
Thanks a ton @aalaap! ☺️

**Pull request merged**